flow/scripts/synth_hier_report.tcl

```diff
@@ -98,16 +100,42 @@ if { [info exist ::env(REPORTS_DIR)] && [file isfile $::env(REPORTS_DIR)/synth_h
   set contents [read -nonewline $fptr]
   close $fptr
   set split_cont [split $contents "\n"]
+  set hierarchy_section 0
+  set module_list {}
+  foreach line $split_cont {
+    if {[regexp {=== design hierarchy ===} $line]} {
+      set hierarchy_section 1
+      continue
+    }
+    if {[regexp {   Number of wires.*} $line ]} {
+      set hierarchy_section 0
+      continue
+    }
+    if { $hierarchy_section == 1 } {
+      if {[regexp { +(\S+) +.*} $line -> module_name]} {
+        lappend module_list $module_name
+        puts "Found module $module_name"
```

Comment:
Is this just for debug?  You will print the module later.

Reply:
With this we can see modules in log file for which area is computed. We can remove this as well.